### PR TITLE
Upgrade to rails 5

### PR DESCRIPTION
**Pull request by RamilyaNigmatullina**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ gem "slim"
 gem "decent_decoration"
 gem "decent_exposure"
 gem "devise"
-gem "draper"
+gem "draper", "3.0.0.pre1"
```
Comment (timurvafin) [inline Gemfile]: I guess we could just lock version in the `Gemfile.lock`

File: Gemfile [hunk 2]
```diff
@@ -42,11 +42,11 @@ gem "rack-mini-profiler", require: false
 gem "responders"
 gem "rollbar"
 gem "seedbank"
+gem "sprockets", "3.6.3"
```
Comment (timurvafin) [inline Gemfile]: And same here

File: bin/server [hunk 3]
```diff
@@ -1,5 +1,3 @@
 #!/usr/bin/env sh
 
-set -e
-
-bin/foreman start
+heroku local
```
Comment (timurvafin) [inline bin/server]: I believe we use `bin/rails server` in rewards.

File: bin/update [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
```
Comment (timurvafin) [inline bin/update]: It will be a little confusing to have `bootstrap` and `update` at the same time

File: bin/rails [hunk 5]
```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-end
-APP_PATH = File.expand_path('../../config/application', __FILE__)
+APP_PATH = File.expand_path('../config/application', __dir__)
```
Comment (timurvafin) [inline bin/rails]: Looks like we missed `spring` here

File: Gemfile [hunk 6]
```diff
@@ -2,21 +2,29 @@ source "https://rubygems.org"
...
+gem "jade-rails"
+gem "mini_racer"
 gem "premailer-rails"
 gem "sass-rails", "~> 5.0.0"
 gem "skim"
-gem "therubyracer", platforms: :ruby
 gem "uglifier", ">= 2.7.2"
 
+source "https://rails-assets.org" do
+  gem "rails-assets-autosize"
```
Comment (timurvafin) [inline Gemfile]: Do you think we need these 3 gems here in rails base?

File: Gemfile [hunk 7]
```diff
@@ -2,21 +2,24 @@ source "https://rubygems.org"
...
 gem "pg"
+gem "rails", "5.0.1"
 
 # assets
 gem "autoprefixer-rails"
 gem "coffee-rails"
 gem "foundation-icons-sass-rails"
 gem "foundation-rails"
-gem "jquery-rails"
+gem "mini_racer"
```
Comment (timurvafin) [inline Gemfile]: Could you please mention this too in the change log?

Comment (RamilyaNigmatullina) [inline Gemfile]: done

File: lib/tasks/assets.rake [hunk 8]
```diff
@@ -6,7 +6,7 @@
 namespace :assets do
   desc "Create .gz versions of assets"
   task gzip: :environment do
-    Dir["#{Rails.root}/public/assets/**/*.{css,html,js,otf,svg,txt,xml}"].each do |file|
+    Dir[Rails.root.join("public", "assets", "**", "*.{css,html,js,otf,svg,txt,xml")].each do |file|
```
Comment (ruslankhaertdinov) [inline lib/tasks/assets.rake]: @RamilyaNigmatullina Should we use `"*.{css,html,js,otf,svg,txt,xml}"` instead of `"*.{css,html,js,otf,svg,txt,xml"`?

Comment (RamilyaNigmatullina) [inline lib/tasks/assets.rake]: thanks!

**Pull request merged**